content/blog/2017/2017-02-02-jenkins-community-survey

```diff
@@ -30,4 +30,3 @@ The push to production has stayed about the same from last year, 61% of responde
 Linux is the platform of choice for builds, favored by 85% of respondents, along with 85% choosing Git as the favored source code repository 
 Half of respondents are deploying applications directly to the cloud, with Amazon Web Services as the favored platform.d
 
-We want to thank everyone for completing the survey - and congrats to Iker Garcia for winning a free pass to Jenkins World 2017 and to Dave Leifer for winning the Amazon gift card.
```

Comment:
Thanking participants is fine, and mentioning winners as well (assuming they are okay with being mentioned by name). Unclear why this was removed.